sdk/identity/identity/samples/AzureCacheForRedis/ioredis.md

```diff
@@ -0,0 +1,274 @@
+## Azure Cache for Redis: Azure AD with ioredis client library
+
+### Table of contents
+
+- [ioredis library](#ioredis-library)
```

Comment:
You can remove this list item. The doc's title already makes it clear that this is the ioredis library.

Reply:
```suggestion
```